zeus/utils/testing.py

```diff
@@ -141,7 +141,7 @@ def end_window(
             return Measurement(
                 time=0.0,
                 gpu_energy={gpu_index: 0.0 for gpu_index in self.gpu_indices},
-                cpu_energy={cpu_index: 0.0 for cpu_index in self.cpu_indices},
+                cpu_energy={},
```

Comment:
`cpu_energy` should be optional like `dram_energy`, shouldn't it? A node may not have a RAPL-compatible CPU.